RentallCarsAPI/Controllers/CarController.cs

```diff
@@ -174,7 +169,7 @@ public IActionResult Delete(Guid id)
                         return BadRequest(response);
                     }
                     response.Succes = true;
```

Comment:
En un futuro podríamos hacer un PR de estilos para agregar una linea a continuación de cada `}`
```suggestion
                    }
                    
                    response.Succes = true;
```